src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,124 @@
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use the exit this page component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+The Exit this page component has several parts, including:
+- a button, that takes the user to another website
+- a secondary link, to give people that use assistive technologies a way to activate the component
+- a loading overlay, to immediately clear any content off the browser until the next website loads
+
+You must use all parts of this component with the 'Exit a page quickly' pattern.
+
+## Adding a button
+
+Position the button component immediately below the header of your page. Its position should be 'sticky', so as the user scrolls down the page, the button remains visible and easy to activate. 
+
+On mobile devices, position the button immediately below the header and make it 'stick' to the top of the screen as the user scrolls down the page.
+
+When the user presses the button, their browser will immediately go to a predetermined page that’s set in the code. The default page is the [homepage of BBC weather](https://www.bbc.co.uk/weather).
+
+### Choosing a different website to load
+
+The component is set to take the user to the [homepage of BBC Weather](https://www.bbc.co.uk/weather). You can change this if there’s a more appropriate site for your users.
+
+Avoid websites that might show personalised pages (such as frequently visited, last visited, or suggested links), as this content could put the user at risk.
+
+## Consider what to do with user session data
+
+You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.
+
+We recommend that services clear the user’s session data once they’ve activated Exit this page, whether they've done so by using the button, the secondary skip link or pressing the shift button 3 times.
+
+## Adding the secondary link
+
+Add the code for the secondary link into the layout file of your service. Place it after your block header.
+
+To activate the secondary link, users will need to either:
+- tab to the Exit this page link
+- press the shift key 3 times
+
+These will perform the same action as pressing the button.
+
+### Showing progress for 'Shift key' presses
+
+To show the user the number of times they’ve pressed the shift button, there’s a series of 3 progress dots under the button, provided as a visual enhancement. Each time the shift key is pressed, one of the dots will be filled.
+
+The user has 5 seconds to press the shift key 3 times. After that, the sequence and progress dots will reset.
+
+This feature requires the user to have Javascript enabled on their browser.
```

Comment:
I wonder if there's a way to frame this to cover all the js features. Putting it just here implies (to me) that only this requires js when js is also controlling the loading overlay.